### ISSUE TITLE
Glossary - لغت

### ISSUE THREAD

**Issue by jabranr** [reactjs/ur.reactjs.org#8]
Adding a list of commonly used translations

| Term | Translation |
| :---: | :---:|
| Apps | ایپس |
| Browser | براوزر |
| Component | جزو |
| Declarative | اعلامیہ |
| Elements | عناصر |
| Editor | مدیر |
| Immutability | ناقابل تغیر |
| Locally | مقامی طور پر |
| Method | طریقہ |
| Overview | جائزہ |
| Parent | والدین |
| Playgrounds | کھیل کا میدان |
| Render | پیش کریں |
| Step-by-step | قدم باقدم |
| Tutorial | سبق |
| Text | متن |
| Try it on Codepen | میں دیکھیں CodePen |
| Value | حاصل  |
| View | ملاحظہ کریں |

Content that does not need to be translated:

| No translation required |
| :-----: |
| API |
| array |
| bundle |
| class |
| constructor |
| context |
| Create React App |
| DOM |
| fragments |
| HTML |
| Hooks |
| JavaScript |
| MDN |
| Tick |
| package | 
| props |
| portal |
| ref |
| React | 
| React Native | 
| strings |
| state |
| Web（Encounter capitalized Web, don't translate）|
| widgets |
| UI |
| key |

> Above list is heavily based on the lists in [Hindi](https://hi.reactjs.org/) and [Arabic](https://ar.reactjs.org/) translation projects.

**Comment by armujahid** [comment]
I think few more words can be moved to "No translation required" list like `Render` and `Parent`.

**Comment by jabranr** [comment]
@armujahid good idea, done 👍

**Comment by arximughal** [comment]
IMO, these words can too move to No Translation Required:
- Declarative
- Editor
- Immutability
- Playgrounds
- Text

Even though these words do have literal meanings in Urdu, but I think their English translation makes more sense to common Urdu speaking folks than direct Urdu translation e.g., Immutability (ناقابل تغیر) is some _next-level_ Urdu grammar, not common to a lot of folks.

**Comment by arximughal** [comment]
Having a dedicated page for such terms where we can explain what these really mean, in Urdu, would also be a good option (though I'm not sure if that's possible).

**Comment by jabranr** [comment]
@arximughal I agree. At times, some of literal meanings do look shocking as those are not part of the daily language. I like the idea of explaining the terms in Urdu while keeping them as they are – so they make sense to the readers.

**Comment by aaftab1441** [comment]
اس پیج کی سرخی فہرست ہونی چاہے
